karma.conf.js

```diff
@@ -1,3 +1,8 @@
+
+/*jslint node: true */
+
+process.env.CHROME_BIN = require( "puppeteer" ).executablePath();
```

Comment:
Can you add a comment that explains why this is needed?

Reply:
Ok:
- Line 2 tells jslint that karma.conf.js is a "node.js" file so "linting" this file goes right 
- Line 4 set the right path to the chrome binary (puppeteer module bundle the chrome binary)

Reply:
I was specifically talking about line 4. What I meant was, in your next patch, can you add a code code comment to the file. It should explain why we need to set CHROME_BIN, when previously it wasn't needed?